File: EnumStoragePoolDataProfile.go
package main

type EnumStoragePoolDataProfile string

const (
	EnumStoragePoolDataProfileraid0  EnumStoragePoolDataProfile = "raid0"
	EnumStoragePoolDataProfileraid1  EnumStoragePoolDataProfile = "raid1"
	EnumStoragePoolDataProfileraid5  EnumStoragePoolDataProfile = "raid5"
	EnumStoragePoolDataProfileraid6  EnumStoragePoolDataProfile = "raid6"
	EnumStoragePoolDataProfileraid10 EnumStoragePoolDataProfile = "raid10"
	EnumStoragePoolDataProfiledup    EnumStoragePoolDataProfile = "dup"
	EnumStoragePoolDataProfilesingle EnumStoragePoolDataProfile = "single"
)

File: pyclient/zeroos/orchestrator/client/EnumStoragePoolDataProfile.py
from enum import Enum


class EnumStoragePoolDataProfile(Enum):
    raid0 = "raid0"
    raid1 = "raid1"
    raid5 = "raid5"
    raid6 = "raid6"
    raid10 = "raid10"
    dup = "dup"
    single = "single"
